controllers/metal3machine_controller.go

```diff
@@ -190,6 +191,28 @@ func (r *Metal3MachineReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	return r.reconcileNormal(ctx, machineMgr)
 }
 
+func patchMetal3Machine(ctx context.Context, patchHelper *patch.Helper, metal3Machine *capm3.Metal3Machine, options ...patch.Option) error {
+
+	// Always update the readyCondition by summarizing the state of other conditions.
+	conditions.SetSummary(metal3Machine,
+		conditions.WithConditions(
+			capm3.AssociateBMHCondition,
+			capm3.Metal3MachineNodeReadyCondition,
+		),
+	)
+
+	// Patch the object, ignoring conflicts on the conditions owned by this controller.
+	options = append(options,
+		patch.WithOwnedConditions{Conditions: []capi.ConditionType{
+			capi.ReadyCondition,
```

Comment:
What does it reflect?

Reply:
Having a standard `ReadyCondition` is considered good practice -- most providers as well as CAPI components use it.
It is a logical AND of all other conditions.